cmd/deploy_test.go

```diff
@@ -102,19 +102,19 @@ func TestDeployRun(t *testing.T) {
 		{
 			"default",
 			[]string{"development", "example.com"},
-			`ansible-playbook deploy.yml -e "env=development site=example.com"`,
+			"ansible-playbook deploy.yml -e env=development site=example.com",
```

Comment:
We need 1 `-e` per ansible variable, i.e: 

```
$ ansible-playbook deploy.yml -e env=development site=typist.tech
ERROR! the playbook: site=typist.tech could not be found

$ ansible-playbook deploy.yml -e env=development -e site=typist.tech
Work as expected
```

Reply:
I believe the culprit is golang is not passing `-e` to `ansible-playbook`

Using trellis-cli 4c5e1eb

```sh-session
$ trellis deploy --extra-vars='a=b' production typist.tech
Running command => ansible-playbook deploy.yml -e "env=production site=typist.tech a=b"

PLAY [Ensure necessary variables are defined] **********************************

TASK [Ensure environment is defined] *******************************************
System info:
  Ansible 2.7.15; Darwin
  Trellis Head
---------------------------------------------------
Environment missing. Use `-e` to define `env`:
ansible-playbook deploy.yml -e "site=<domain> env=<environment>"

fatal: [localhost]: FAILED! => {"changed": false}
	to retry, use: --limit @/Users/rufus/Code/typist/trellis/deploy.retry

PLAY RECAP *********************************************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1
```

However, if I run the generated command directly:

```sh-session
$ ansible-playbook deploy.yml -e "env=production site=typist.tech a=b"
It works
```

Reason is unknown.

Reply:
😕 gotta dig into this more I guess

Thanks for testing

Reply:
Wait I thought you were testing this branch? 4c5e1eb is master which has this known bug. Can you try with this branch? It's working for me

Reply:
Oh............ I need coffee.
5a66436 works.
Sorry for wasting time